monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -498,11 +502,32 @@ def train_single_model(self, config: Any, fold: int, gpu_id: int = 0, **kwargs:
                     - disable_checkpointing: True to disable checkpointing. Ideal for testing things out and you
                         don't want to flood your hard drive with checkpoints. Default: False.
         """
-        os.environ["CUDA_VISIBLE_DEVICES"] = f"{gpu_id}"
+        if isinstance(gpu_id, str):
+            gpu_id = tuple(map(int, gpu_id.replace('"', "").split(",")))
+
+        if isinstance(gpu_id, tuple):
+            if len(gpu_id) > 1:
+                gpu_ids_str = ""
+                for _i in range(len(gpu_id)):
+                    gpu_ids_str += f"{gpu_id[_i]},"
+                os.environ["CUDA_VISIBLE_DEVICES"] = gpu_ids_str[:-1]
```

Comment:
Does the following work for L510-513?
os.environ["CUDA_VISIBLE_DEVICES"] = ",".join([str(x) for x in gpu_id]

Reply:
I have eliminated support for string input.

Reply:
It was a suggestion to simplify the follow lines (originally L510-513)
```
gpu_ids_str = ""
for _i in range(len(gpu_id)):
    gpu_ids_str += f"{gpu_id[_i]},"
os.environ["CUDA_VISIBLE_DEVICES"] = gpu_ids_str[:-1]
```
can be replaced by:
```
os.environ["CUDA_VISIBLE_DEVICES"] = ",".join([str(x) for x in gpu_id])
```
It is minor and please feel free to ignore

Reply:
@dongyang0122